src/SourceBuild/content/repo-projects/nuget-client.proj

```diff
@@ -4,8 +4,14 @@
     <NuGetKeyFilePath>$(KeysDir)NuGet.Client.snk</NuGetKeyFilePath>
     <DeterministicBuildOptOut>true</DeterministicBuildOptOut>
 
+    <!-- Don't pass any build actions because the repo doesn't support them. -->
+    <BuildActions></BuildActions>
```

Comment:
This worked before so I wonder what changed?

Reply:
I only worked before because the .sh script is just set up to swallow anything starting with `-`